<launch>
	<node name="dsl_grid3d" pkg="dsl_grid3d" type="dsl_grid3d_node" output="screen">
		<param name="mesh_filename" type="string" value="$(find dsl_grid3d)/data/smallmap_rot.stl"/>
		<param name="use_textured_mesh" type="bool" value="false"/>
		<param name="cells_per_meter" type="double" value="2.0"/>
	</node>
</launch>
